tests/expense.test.js

```diff
@@ -53,7 +54,7 @@ describe('Expense', () => {
 
     it('should return 400 if user not found', async() => {
       const expenseData = {
-        userId: 1,
```

Comment:
Why do you change it?

Reply:
try to understand why server not shutted down, return it to initial state now